base/float.jl

```diff
@@ -858,6 +860,19 @@ julia> floatmin(Float32)
 julia> floatmin()
 2.2250738585072014e-308
 ```
+
+# Extended help
+
+The smallest positive number without this restriction is
```

Comment:
As written, this paragraph's "this" referred to something. But in moving it around, now it's dangling.

Reply:
I think referring the "non-subnormal" solves it as well